apps/dashboard/src/main/java/com/akto/listener/InitializerListener.java

```diff
@@ -1000,49 +1010,70 @@ private String getUpdateDeploymentStatusUrl() {
     }
 
     public static void saveTestEditorYaml() {
-        List<String> templatePaths = new ArrayList<>();
-        try {
-            templatePaths = convertStreamToListString(InitializerListener.class.getResourceAsStream("/inbuilt_test_yaml_files"));
-        } catch (Exception e) {
-            loggerMaker.errorAndAddToDb(String.format("failed to read test yaml folder %s", e.toString()), LogDb.DASHBOARD);
-        }
+        Map<String, String> templates = null;
```

Comment:
initialise this properly. 
Map<String, String> templates = new HashMap
Refer to other places in code